lib/dataset/query.js

```diff
@@ -338,6 +338,50 @@ define({
             return this.mergeOptions({distinct: args});
         },
 
+        /*   {
+         *     division: db.from('division').one();
```

Comment:
Another benefit of accepting functions would be the ability to pass the data/model in as an argument

```javascript
Authors
//will work for multiples
.eager({
    "books": user => Books.filter({authorId: user.id}).all(),    
})
.all()
```